lib/travis/api/v3/models/json_slice.rb

```diff
@@ -2,7 +2,7 @@
 
 module Travis::API::V3
   class Models::JsonSlice
-    include Virtus.model, Enumerable, Models::JsonSync
+    include Virtus.model, Enumerable, Models::JsonSync, ActiveModel::Model
```

Comment:
Seems heavy! Is this to make the square bracket assignment work?

Reply:
No, it's for the validations! It turns out, `Virtus.model` does type coercion but not validation, they recommend to use something else for that, such as dry-validations. I also saw some examples online using ActiveModel together for that and since that's already a dependence... that said, I had planned to look into a more minimal approach (there's an `ActiveModel::Validations`, for starters) but I forgot later :roll_eyes: I'll see if we can avoid mixing in the whole `ActiveModel::Model`.

Reply:
Seems to work exactly the same with `ActiveModel::Validations`. Good catch, I think this a [much smaller API to include](https://api.rubyonrails.org/classes/ActiveModel/Validations.html), with zero overlap with `Virtus.model`. Smaller enough?